coordinator/internal/logic/provertask/batch_prover_task.go

```diff
@@ -31,6 +31,7 @@ type BatchProverTask struct {
 
 	batchAttemptsExceedTotal prometheus.Counter
 	batchTaskGetTaskTotal    *prometheus.CounterVec
+	batchTaskGetTaskProver   *prometheus.CounterVec
```

Comment:
Why metrics have record the metrics in getTaskController, why record here again

Reply:
Same concern, is it for the total requests and the real effective requests? @amoylan2

Reply:
When a prover call the get_task api, it may not get task successfully.
So the metrics in controller are targeting whether a prover is "online", the metrics in logic are targeting "successfully get a task" or "not idle".